test/index.spec.ts

```diff
@@ -116,7 +116,7 @@ describe('Public API', () => {
         artifactName: 'electron.d.ts',
       });
       expect(await fs.pathExists(dtsPath)).toEqual(true);
-      expect(path.basename(dtsPath)).toEqual('v2.0.9-electron.d.ts');
```

Comment:
This is deliberate, all other artifacts have the version name in them.  If this artifact does not it will be duplicated and overwritten right?

Reply:
It can be only overwritten if the downloader called with `force: true`.
`electron.d.ts` files for other Electron versions/mirrors will be downloaded to other folders.

The goal of this PR is to make sure every downloaded file is saved to the cache with a unique path. A folder name serves as a unique name, and a file names does not matter for caching.